pylint/lint/run.py

```diff
@@ -21,7 +21,7 @@
 def _cpu_count() -> int:
     """Use sched_affinity if available for virtualized or containerized environments."""
     sched_getaffinity = getattr(os, "sched_getaffinity", None)
-    # pylint: disable=not-callable,using-constant-test
+    # pylint: disable=not-callable,using-constant-test,useless-suppression
```

Comment:
This is unsolvable atm. The `pre-commit` env does have `sched_getaffinity` so this will always create some sort of error.